test/es6-imports.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env jest */
+
+// Dependencies
+const runProgram = require('./run-program');
+
+// Initialize
+const entryFile = `${__dirname}/programs/pizza-ordering/cli/entry.js`;
```

Comment:
You're not referencing the correct entry point:

```suggestion
const entryFile = `${__dirname}/programs/es6-imports/cli/entry.js`;
```